src/data/hymnalsConfig.tsx

```diff
@@ -0,0 +1,124 @@
+export type HymnalConfig = {
+    language: string;
+    title: string;
+    key: string;
+};
+
+export const getHymnalFileUrl = (language: string) =>
+    `https://raw.githubusercontent.com/TinasheMzondiwa/cis-hymnals/main/${language}.json`;
+
+/**
+ * The config data was copied from this file:
+ *
+ * https://github.com/TinasheMzondiwa/cis-hymnals/blob/main/config.json
+ *
+ * Instead of including it hardcoded, as it is now, we could alternatively
+ * fetch it at runtime
+ */
+export const HYMNALS_CONFIG: HymnalConfig[] = [
```

Comment:
@TinasheMzondiwa can you confirm if all these hymns are ready ?  Maybe we can add another field per hymn to indicate that's it's not ready for public or maybe not incomplete.

Reply:
All hymns in the main directory should be ready